meson.build

```diff
@@ -1437,7 +1437,7 @@ if vaapi_wayland['use']
     features += vaapi_wayland['name']
 endif
 
-if vaapi_wayland['use'] and memfd_create
+if vaapi_wayland['use'] and memfd_create and wayland['deps'][2].version().version_compare('>= 1.25')
```

Comment:
TODO: before this is merged the correct `wayland-protocols` version must be used